src/app/(auth)/styles.tsx

```diff
@@ -42,17 +47,16 @@ export const PCentered = styled(P)`
   text-align: center;
 `;
 
-export const AuthHeading = styled.h1`
-  color: ${COLORS.blueDarker};
-  font-size: 48px;
+export const AuthHeading = styled(H1)`
+  font-size: 3rem;
   font-style: normal;
-  font-weight: 600;
 `;
 
 export const AuthSubHeading = styled.h2`
```

Comment:
do we want this to be a `styled(H2)`? If so, we can remove the redundant font-size declaration, which would be inherited from H2.